src/tasks.cr

```diff
@@ -5,4 +5,10 @@ module PlaceOS::Tasks
   include Database
   include Entities
   include Initialization
+
+  PROD = (ENV["ENV"]? || ENV["SG_ENV"]?) == "production"
```

Comment:
I may be mistaken, but think this result in the compile time values for these env vars, whereas it looks like runtime is desired.

Reply:
```crystal
CONSTANT = ENV["FOO"]? || "runtime"
puts CONSTANT
```

```bash
$ crystal build test.cr
$ ./test
runtime
$ FOO=hello ./test
hello
```

Reply:
Oh nice! My mental model for that was well and truly broken.

Reply:
[This is the compile time env lookup](https://crystal-lang.org/api/0.35.1/Crystal/Macros.html#env(name):StringLiteral|NilLiteral-instance-method)